sdk/communication/azure-communication-administration/README.md

```diff
@@ -272,6 +272,29 @@ PhoneNumberClient phoneNumberClient = createPhoneNumberClient();
 phoneNumberClient.configureNumber(phoneNumber, pstnConfiguration);
 ```
 
+## Long Running Operations
+
+The Phone Number Client supports a variety of long running operations that allow indefinite polling time to the functions listed down below. Because these functions may take a long time to resolve, the use of long running operations is encouraged.
```

Comment:
encouraged? I thought that is the only way to access these method..

Reply:
Didn't know that but makes sense.

Reply:
Would it make sense to then remove the normal createSearch example from the README? @ankitarorabit

Reply:
Yes, I think that should be done.. thanks for pointing out.